package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "workerd",
```

Comment:
Can we call this something else/put it somewhere else? The `workerd` package name is already in use in the `npm` directory